pyterrier/text.py

```diff
@@ -176,14 +177,23 @@ def sliding( text_attr='body', length=150, stride=75, join=' ', prepend_attr='ti
         prepend_attr(str): whether another document attribute, such as the title of the document, to each passage, following [Dai2019]. Defaults to 'title'. 
         title_attr(str): what is the name of the dataframe attribute containing the title the document to be split into passages.
             Default is 'title'. Only used if prepend_title is set to True.
+        tokenizer(obj): which model to use for tokenizing. the object must have a `.tokenize(str) -> list[str]` method for tokenization.
```

Comment:
We should be more specific about the requirements - it must have a convert_tokens_to_string() method too